.github/workflows/deploy.yml

```diff
@@ -5,16 +5,19 @@ name: Deploy
 run-name: Deploy ${{ github.ref_name }} to ${{ inputs.environment || 'dev' }}
 
 on:
-  # !! Uncomment the following lines once you've set up the dev environment and ready to turn on continuous deployment
-  # push:
-  #   branches:
-  #     - 'main'
-  #   paths:
-  #     - 'frontend/**'
-  #     - 'bin/**'
-  #     - 'infra/**'
```

Comment:
we want to leave this commented out

Reply:
Why is that? We have a dev environment set up

Reply:
sorry i totally thought i was reviewing a Pr in a different repo